crates/sui-bridge/src/config.rs

```diff
@@ -87,38 +108,53 @@ impl BridgeNodeConfig {
     pub async fn validate(
         &self,
     ) -> anyhow::Result<(BridgeServerConfig, Option<BridgeClientConfig>)> {
+        if !is_route_valid(
+            BridgeChainId::try_from(self.sui.sui_bridge_chain_id)?,
+            BridgeChainId::try_from(self.eth.eth_bridge_chain_id)?,
+        ) {
+            return Err(anyhow!(
+                "Route between Sui chain id {} and Eth chain id {} is not valid",
+                self.sui.sui_bridge_chain_id,
+                self.eth.eth_bridge_chain_id,
+            ));
```

Comment:
would it make sense at some point to define an `Error` struct/module for the bridge?
We could have a classic ID that makes sense besides the strings.
I suppose it could make even testing better?

Reply:
we do have `BridgeResult` and `BridgeError`, i chose to use `anyhow` here cuz there can be other types of errors in this code path (when starting a node). But in the "core" of bridge node, you will see `BridgeResult/Error` everywhere